### DISCUSSION TITLE
Explicit Music should have [E] instead of complete "explicit"

### DISCUSSION THREAD

**Discussion by RSickenberg** [lowlighter/metrics#1673] [Ideas]
<img width="238" alt="image" src="https://github.com/user-attachments/assets/52fd587d-1cf1-482e-b248-56d61c620b08">
Here is what is rendered on my side.

**Comment by RSickenberg** [comment]
My suggestion, would to scrap "Explicit Content" from this line: https://github.com/lowlighter/metrics/blob/master/source/plugins/music/index.mjs#L116

**Comment by RivenSkaye** [comment]
Moving over here so as not to turn #1674 into a discussion.
What would be wrong with adding [E] after the artist name? Could even plant an en-dash in between there to visually split them as well. I'm not talking about a plain E, I was specifically suggesting it _with the square braces_ to make it visually stand out

**Reply by RSickenberg** [reply]
Regarding the [E] *after* the artist states that's the artist who's explicit but not the title IMHO.
<img width="613" alt="image" src="https://github.com/user-attachments/assets/6266b626-8bf4-4193-bad0-a19af39bb7ea" />
Apple add the [E] after the title.

I'm not against the square, in fact it would be an acceptance criterion for me. But not at the artist level :)

**Reply by RivenSkaye** [reply]
Spotify does this, which I think is reasonable because of the song title being in a larger font size
![image](https://github.com/user-attachments/assets/7fa14c74-3925-4b5a-adb0-109aa4edae1e)

(the highlight is because I knew they'd be easy to find both explicit and non-explicit for)

**Reply by RSickenberg** [reply]
I see yes @RivenSkaye.
It's something similar but two UI different.

On Spotify we can argue that the [E] is under the title but I maintain that's the song that's explicit, not the artist.
